js/views/install_view.js

```diff
@@ -32,8 +32,9 @@
             this.connect();
             this.on('disconnected', this.reconnect);
 
-            if (Whisper.Registration.everDone() || options.startStep) {
-                this.selectStep(options.startStep || Steps.SCAN_QR_CODE);
+            // Keep data around if it's a re-link, or the middle of a light import
+            if (Whisper.Registration.everDone() || options.postImport) {
+                this.retainData = true;
             }
```

Comment:
Could we shorten this to `this.retainData = Whisper.Registration.everDone() || options.postImport;`? Could we also rename `retainData` to `shouldRetainData` to indicate it’s a boolean property rather than a method (verb)?

Reply:
Alright, I'm still getting used to this `should`/`is` naming scheme. Yep, we can shorten this.